src/app/SharpRaven/IRavenClient.cs

```diff
@@ -88,14 +88,29 @@ public interface IRavenClient
         /// </value>
         TimeSpan Timeout { get; set; }
 
-
+        
         /// <summary>Captures the specified <paramref name="event"/>.</summary>
         /// <param name="event">The event to capture.</param>
         /// <returns>
         /// The <see cref="JsonPacket.EventID" /> of the successfully captured <paramref name="event" />, or <c>null</c> if it fails.
         /// </returns>
         string Capture(SentryEvent @event);
 
+        /// <summary>
+        /// Captures the <see cref="Breadcrumb"/> for tracking.
+        /// </summary>
+        /// <param name="breadcrumb">The <see cref="Breadcrumb" /> to capture.</param>
+        void AddTrail(Breadcrumb breadcrumb);
+
+        /// <summary>
+        /// Restart the capture of the <see cref="Breadcrumb"/> for tracking.
+        /// </summary>
+        void RestartTrails();
```

Comment:
Similiar the AddTrail in PythonAPI uses this way: raven.breadcrumbs.record(**options).  https://docs.getsentry.com/hosted/clients/python/breadcrumbs/#manually-emitting-breadcrumbs

About restartTrails was more new idea. I use in production scenario when the support analyst to clear the log for simulate the problem and i can to have a clean log.